server/api/post_action.go

```diff
@@ -96,15 +96,15 @@ func (api *api) postActionRespond(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	p, err := api.PluginAPI.GetPost(postID)
-	if err != nil {
-		utils.SlackAttachmentError(w, "Error: Failed to update the post: "+err.Error())
+	p, appErr := api.PluginAPI.GetPost(postID)
+	if appErr != nil {
+		utils.SlackAttachmentError(w, "Error: Failed to update the post: "+appErr.Error())
 		return
 	}
```

Comment:
Please double check that the logic is correct now. With `err` L113 makes little sense.

Reply:
@hanzei that looks fine for me